robolectric-shadows/shadows-core/src/main/resources/org/robolectric/shadows/ShadowAccessibilityNodeInfo.java.vm

```diff
@@ -265,108 +346,205 @@ public class ShadowAccessibilityNodeInfo {
+  }
+
+  @Implementation
+  public boolean isContentInvalid() {
+    return ((propertyFlags & CONTENT_INVALID_MASK) != 0);
+  }
+
+  @Implementation
+  public void setDismissable(boolean dismissable) {
+    propertyFlags = (propertyFlags & ~DISMISSABLE_MASK) |
+        (dismissable ? DISMISSABLE_MASK : 0);
+  }
+
+  @Implementation
+  public boolean isDismissable() {
+    return ((propertyFlags & DISMISSABLE_MASK) != 0);
+  }
+
+  @Implementation
+  public void setCanOpenPopup(boolean opensPopup) {
+    propertyFlags = (propertyFlags & ~CAN_OPEN_POPUP_MASK) |
+        (opensPopup ? CAN_OPEN_POPUP_MASK : 0);
+  }
+
+  @Implementation
+  public boolean canOpenPopup() {
+    return ((propertyFlags & CAN_OPEN_POPUP_MASK) != 0);
+  }
+#end
+
   public void setTextSelectionSetable(boolean isTextSelectionSetable) {
-    actionFlags = (actionFlags & ~TEXT_SELECTION_SETABLE_MASK) | 
+    propertyFlags = (propertyFlags & ~TEXT_SELECTION_SETABLE_MASK) |
         (isTextSelectionSetable ? TEXT_SELECTION_SETABLE_MASK : 0);
   }
 
   @Implementation
   public void setClickable(boolean isClickable) {
-    actionFlags = (actionFlags & ~CLICKABLE_MASK) | (isClickable ? CLICKABLE_MASK : 0);
+    propertyFlags = (propertyFlags & ~CLICKABLE_MASK) | (isClickable ? CLICKABLE_MASK : 0);
   }
 
   @Implementation
   public void setLongClickable(boolean isLongClickable) {
-    actionFlags =
-        (actionFlags & ~LONGCLICKABLE_MASK) | (isLongClickable ? LONGCLICKABLE_MASK : 0);
+    propertyFlags =
+        (propertyFlags & ~LONGCLICKABLE_MASK) | (isLongClickable ? LONGCLICKABLE_MASK : 0);
   }
 
   @Implementation
   public void setFocusable(boolean isFocusable) {
-    actionFlags = (actionFlags & ~FOCUSABLE_MASK) | (isFocusable ? FOCUSABLE_MASK : 0);
+    propertyFlags = (propertyFlags & ~FOCUSABLE_MASK) | (isFocusable ? FOCUSABLE_MASK : 0);
   }
 
   @Implementation
   public void setFocused(boolean isFocused) {
-    actionFlags = (actionFlags & ~FOCUSED_MASK) | (isFocused ? FOCUSED_MASK : 0);
+    propertyFlags = (propertyFlags & ~FOCUSED_MASK) | (isFocused ? FOCUSED_MASK : 0);
   }
 
   @Implementation
   public void setScrollable(boolean isScrollable) {
-    actionFlags = (actionFlags & ~SCROLLABLE_MASK) | (isScrollable ? SCROLLABLE_MASK : 0);
+    propertyFlags = (propertyFlags & ~SCROLLABLE_MASK) | (isScrollable ? SCROLLABLE_MASK : 0);
   }
 
   public void setPasteable(boolean isPasteable) {
-    actionFlags = (actionFlags & ~PASTEABLE_MASK) | (isPasteable ? PASTEABLE_MASK : 0);
+    propertyFlags = (propertyFlags & ~PASTEABLE_MASK) | (isPasteable ? PASTEABLE_MASK : 0);
   }
 
   @Implementation
   public void setEditable(boolean isEditable) {
-    actionFlags = (actionFlags & ~EDITABLE_MASK) | (isEditable ? EDITABLE_MASK : 0);
+    propertyFlags = (propertyFlags & ~EDITABLE_MASK) | (isEditable ? EDITABLE_MASK : 0);
     /* If a shadow node is editable, then its text is also selectable. However, if a shadow node is
```

Comment:
I think we should take this behavior out.  This is framework behavior, but not AccessibilityNodeInfo behavior, so I don't think it should be accounted for here.